01 - Community/People/🗂️ People.md

```diff
@@ -517,6 +517,7 @@ Overviews of all the work from notable People like Plugins Developers, Theme Des
 -  [[01 - Community/People/twentytwokhz|twentytwokhz]]
 -  [[01 - Community/People/twibiral|twibiral]]
 -  [[01 - Community/People/tylernguyen|tylernguyen]]
+-  [[01 - Community/People/universvm|Leonardo Castorina]]
```

Comment:
For future reference, these lists of files are machined-generated by script, and don't need to be updated manually.

You don't need to remove it now, though. We can deal with when we merge.